### DISCUSSION TITLE
Activity overview graph

### DISCUSSION THREAD

**Discussion by sitiom** [lowlighter/metrics#248] [Ideas]
Whatever this thing is:
![Activity overview graph](https://user-images.githubusercontent.com/56180050/115655431-e392b880-a365-11eb-9919-1cc93b77178e.png)

**Comment by lowlighter** [comment]
Chartist doesn't support radar graphs (https://github.com/gionkunz/chartist-js/issues/390) so it's not possible atm 😕 

Maybe switching to another SVG graph generator which also works on nodejs would be a solution so it would become possible to implement this, but it would also require additional work

**Comment by jayvdb** [comment]
chartist looks dead. https://github.com/gionkunz/chartist-js/branches https://github.com/gionkunz/chartist-js/issues/1310
Might be time to choose a new one anyway. All of the alternatives appear to support radar/spider graphs.

**Reply by lowlighter** [reply]
Yeah, will eventually switch to d3 since it'll be included with #1137
